domains/notjunar.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "NotJunar"
```

nvm, missing a comma here
```suggestion
        "username": "NotJunar",
```